src/nvim/buffer_defs.h

```diff
@@ -778,7 +778,10 @@ struct file_buffer {
                                  * normally points to this, but some windows
                                  * may use a different synblock_T. */
 
-  signlist_T *b_signlist;       /* list of signs to draw */
+  signlist_T *b_signlist;       // list of signs to draw
+  int b_signcols_max;           // cached maximum number of signs in a line
+  int b_signcols;               // last calculated maximum number of signs
+                                // in a line
```

Comment:
it says `b_signcols` is the "maximum". That seems confusing. Isn't it the "actual number"? So the docstring could be:

    // last calculated number of sign columns

"in a line" is redundant since we're talking about columns.

Reply:
Sure. Fixed the docstring.